.cirrus.yml

```diff
@@ -82,6 +82,8 @@ build_task:
   build_script:
     - git submodule update --init
     - source cirrus-env BUILD
+    - mvn -f python-frontend/pom.xml protobuf:compile-python@generate-protobuf-python-sources
+    - tox -c python-frontend/typeshed_serializer/tox.ini
```

Comment:
If you run tox like that, doesn't it run both the tests and the serialization? I'm not sure what is the exact command to only run the tests, though...

Reply:
Indeed thanks!
After some manual test, I believe that running directly `python -m pytest --cov=serializer --cov-report xml:cov.xml --ignore=resources` will only execute the tests